package.json

```diff
@@ -1829,6 +1829,37 @@
                     "separator": "\\"
                 }
             }
+        ],
+        "walkthroughs": [
+            {
+                "id": "getStarted",
+                "title": "%AWS.walkthrough.gettingStarted.title%",
+                "description": "%AWS.walkthrough.gettingStarted.description%",
+                "when": "!config.aws.profile",
+                "steps": [
+                    {
+                        "id": "accessKeys",
+                        "title": "%AWS.walkthrough.gettingStarted.accessKeys.title%",
+                        "description": "%AWS.walkthrough.gettingStarted.accessKeys.description%",
+                        "media": {
+                            "image": "resources/walkthrough/iamCredentials.png",
+                            "altText": "Empty image"
```

Comment:
We should probably have something more descriptive here, I'm assuming this would be used by screen readers, etc. Should this also be localized/can it be?

Reply:
Yeah 'altText' refers to accessibility in this case, and it should be localized.

Reply:
I wouldn't be surprised if VS Code lets you pull a remote image (in which this is the placeholder). But yeah, anything is better than this.

Reply:
You're both correct. And interestingly they do allow remote images here, though in this case I'm going to embed them. Added some more meaningful text sourced through the NLS file.